docs/articles/documentation/test-api/assertions/assertion-api.md

```diff
@@ -35,7 +35,7 @@ await t.expect( actual ).eql( expected, message, options );
 
 Parameter              | Type                                              | Description
 ---------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------
-`actual`             | Any type | A comparison value.
+`actual`             | Value &#124; Object &#124; Selector's Property &#124; ClientFunction | A comparison value. If you pass a [selector's property](../selecting-page-elements/selectors.md#define-assertion-actual-value) or a [client function](../obtaining-data-from-the-client.md) promise, the [Smart Assertion Query Mechanism](README.md#smart-assertion-query-mechanism) is activated and the assertion automatically waits until the comparison value is obtained.
```

Comment:
You said [above](https://github.com/DevExpress/testcafe/pull/1783/files#diff-7ca810eb42a80a987817c213a8b94b34R136) that assertion receives a client function *promise* .
Should we write `ClientFunction` -> `ClientFunction promise`?
Why we need to specify `Value` and `Object`, `value` doesn't include `Object` type?

Reply:
Agree about promise. Should we leave `Value` or `Object`? AFAIK, they are all objects so we should leave `Object`, but you say that value includes object?

Reply:
`Value` is not a type, so I don't know what is this) 
May be write smth like this: `Any type (include Selector's Property and ClientFunction promise)`

Reply:
Hmm, we were going to get rid of `Any type`. Maybe we just remove `Value`?

Reply:
I'm not sure that it's intuitively clear that numbers, strings etc.. are `Objects` (but it's technically true in javascript). 
`Any Type, Selector's Property, ClientFunction promise` look ok for me. or `Any Value...`
`Value` it's not bad too.